h/templates/layouts/base.html

```diff
@@ -10,7 +10,13 @@
       {% endfor -%}
     {% endblock %}
 
-    <title>{% block title %}Hypothesis{% endblock %}</title>
+    <title>
+      {%- block title -%}
+        {%- block page_title %}{% endblock -%}
+        {%- if self.page_title() %} | {% endif -%}
```

Comment:
This is jinja stuff that I don't know about. Can you just clarify for me what's happening here? Is `self` the current template and therefore `self.page_title()` renders that block? Would it be enough to just say `self.page_title` without the function application if we just want to check that the block exists?